JSDemos/Demos/PivotGrid/LayoutCustomization/description.md

```diff
@@ -1 +1,15 @@
-The PivotGrid component supports multiple customizations allowing you to&nbsp;provide the look and feel that will satisfy any user. This demo illustrates only a&nbsp;small part of&nbsp;the available customizations. Toggle the check boxes above the PivotGrid to&nbsp;change the layout at&nbsp;runtime.
\ No newline at end of file
+PivotGrid allows users to customize layout and orientation of elements that annotate data. These elements are called [headers](/Documentation/Guide/UI_Components/PivotGrid/Visual_Elements/#Headers). They can be row, column, or data headers. You can customize row and data headers. Use the following properties to configure headers: 
```

Comment:
`showTotalsPrior` is not about headers.

Reply:
I meant that the intro shouldn't focus on headers because it's only logical to include showTotalsPrior in the list.

Now you have another problem: four sentences in a row end with the same word "headers".

> PivotGrid allows users

It allows "you", the developer.